### PR TITLE
Support transfer with twin Id

### PR DESCRIPTION
**Pull request by AlaaElattar**

### CODE REVIEW

**Comment by AlaaElattar** [comment]
Screenshot from 2023-07-17 12-02-58
Screenshot from 2023-07-17 12-03-04
Screenshot from 2023-07-17 12-03-15
1/97b75340-5844-426e-8b61-8bbd3a1e0aeb)

File: packages/dashboard/src/portal/views/Transfer.vue [hunk 1]
```diff
@@ -3,50 +3,115 @@
...
+                <v-combobox
+                  v-model="receptinTwinId"
+                  :items="accountTwinIds"
+                  dense
+                  filled
+                  @keydown="setValue"
+                  label="Recipient:"
+                  :rules="[
+                    () => !!receptinTwinId || 'This field is required',
+                    () => transferTwinIdCheck() || 'invalid address',
```
Comment (AhmedHanafy725) [inline packages/dashboard/src/portal/views/Transfer.vue:72]: this field should have also validation to be only positive integer numbers

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]
in case the twin doesn't exist, no errors and the `submit` button is active but does nothing

**Comment by AhmedHanafy725** [comment]
the validation sometimes works and sometimes not 

https://github.com/threefoldtech/tfgrid-sdk-ts/assets/41957921/584e1a06-7c91-42f4-be09-d67e8c87a4b0

File: packages/dashboard/src/portal/views/Transfer.vue [hunk 2]
```diff
@@ -135,23 +228,72 @@ export default class TransferView extends Vue {
...
     ).catch(err => {
       this.$toasted.show(err.message);
-      this.loadingTransfer = false;
+      this.loadingTransferAddress = false;
     });
   }
 
+  async transferTFTWithTwinID() {
+    const twinDetails = await this.queryClient.twins.get({ id: parseInt(this.receptinTwinId) });
+    const injector = await web3FromAddress(this.$store.state.credentials.account.address);
```
Comment (AhmedHanafy725) [inline packages/dashboard/src/portal/views/Transfer.vue:245]: it's still using the extension while it's gone on the development branch, so please merge the development branch into this branch and use the mnemonic instead

**Pull request closed without merging**